service/src/main/kotlin/no/nav/su/se/bakover/service/revurdering/RevurderingServiceImpl.kt

```diff
@@ -170,6 +172,20 @@ internal class RevurderingServiceImpl(
             if (!it.tidslinjeForVedtakErSammenhengende()) return KunneIkkeOppretteRevurdering.TidslinjeForVedtakErIkkeKontinuerlig.left()
         }
 
+        gjeldendeVedtaksdata.periode.tilMånedsperioder()
+            .mapNotNull { gjeldendeVedtaksdata.gjeldendeVedtakPåDato(it.fraOgMed) }
+            .filterIsInstance<Vedtak.EndringIYtelse.OpphørtRevurdering>()
+            .any {
```

Comment:
Kunne brukt en foreach og if her for å tydeliggjøre at resultatet ikke brukes til noe